cmd/template/cluster/runner.go

```diff
@@ -108,12 +115,12 @@ func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) err
 
 	switch r.flag.Provider {
 	case key.ProviderAWS:
-		err = provider.WriteAWSTemplate(output, config)
+		err = provider.WriteAWSTemplate(c.K8sClient, output, config)
```

Comment:
I'd say let's pass the `ctx` from here into `WriteCAPATemplate` and `WriteCAPZTemplate` instead of recreating it there.

Reply:
Done.